src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -284,14 +283,6 @@ private static void validateTableAccess(Client engineClient, JSONObject mapping,
             } catch (DataServiceException e) {
                 databaseTableErrorList.add(String.format("Couldn't validate access to Database '%s' Table '%s', with exception '%s'", database, table, ExceptionUtils.getStackTrace(e)));
             }
-            if (hasAccess) {
-                try {
```

Comment:
actually String mappingName = mapping.getString(MAPPING); throws JSONException when the mapping is not present. But I get your concern I will rather use optString instead of getString.